packages/node/src/protocol/install.ts

```diff
@@ -325,6 +325,7 @@ function computeStateChannelTransition(
     /* latestState */ initialState,
     /* latestVersionNumber */ 0,
     /* defaultTimeout */ defaultTimeout,
+    /* outcomeType */ twoPartyOutcomeInterpreterParams ? OutcomeType.TWO_PARTY_FIXED_OUTCOME : OutcomeType.COIN_TRANSFER,
```

Comment:
Why not just make `outcomeType` a `get` property on `AppInstance` with this logic?